accessibility/README.md

```diff
@@ -0,0 +1,495 @@
+  * See [this blog post from 2020](https://gomakethings.com/revisting-aria-label-versus-a-visually-hidden-class/)
+* Examples of using this are similar to adding a `<span class="visually-hidden">` element
+
+```vue
+<button
+  type="button"
+  aria-label="Toggle Menu"
+  @click="openMenuDrawer"
+>
+  <icon-menu />
+</button>
+
+<a href="/account" aria-label="Account">
+  <icon-account />
+</a>
+```
+
+[ꜛ Back to TOC](#table-of-contents)
+
+## Alt Text
+* Alt text is extremely useful for ensuring that images on the site can be used by people with various disabilities.
+* Images can be categorised in different ways and there are some different recommendations for alt text depending on the type of image.
+* The W3C Web Accessibility Initiative has some good resources around Images:
+  * [Images Tutorial](https://www.w3.org/WAI/tutorials/images/) - Outlines the different purposes an image might have and recommendations for adding accessible text
+  * [`alt` Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) - Provides a helpful decision tree on how to handle different situations with images, including when an empty `alt` attribute is appropriate. 
+
+[ꜛ Back to TOC](#table-of-contents)
+
+## Element IDs
+
+* A variety of accessibility code and functionality (such as the `label` element's `for` attribute, `aria-labelledby` and other `aria-` attributes) use the `id` attribute to reference other elements
+  * For example, linking a checkbox input with a label using `id`/`for` allows the user to click on the label to toggle the checkbox
+  * This helps make the input easier to trigger with a mouse, and also makes it possible to style custom inputs (for example custom checkboxes) by applying styles to the `<label>` element
+* When writing components that use the `id` attribute - ensure that the `id` will always be unique
+  * Avoid hard coding an id for an element unless you know it will only be used once
+  * This is particularly important when the component is designed to be re-used, for example a swatch or a line item component
+  * Unique ids can be generated using passed in data, and additionally namespaces if needed
+  * For section type Canvas components - you could also pass the Shopify `section.id` as a prop and use that in your element id
+
+Example of a dynamic id based on unique prop data: 
+```vue
+<template>
+  <div class="quantity-selector">
+    <label
+      :for="`quantity-selector-input-${lineItem.key}`"
+      class="visually-hidden"
+    >
+      Quantity
+    </label>
+
+    <input
+      :id="`quantity-selector-input-${lineItem.key}`"
+      type="number"
+      value="1" 
+    >
+  </div>
+</template>
+<script>
+export default {
+  name: 'QuantitySelector',
+  
+  props: {
+
+    // Cart line item data from Shopify
```

Comment:
Shouldn't have a newline before it as it's an inline comment. Only block comments require them.